build.sbt

```diff
@@ -7,8 +7,9 @@ version := "0.0.1"
 scalaVersion := "2.12.11"
  
 libraryDependencies ++= Seq(
+  "com.tresys" % "dfdl-nitf-core" % "0.0.1",
   "com.mitre" % "dfdl-jpeg" % "0.0.1",
```

Comment:
If the jpeg stuff goes under /src/test, then this dependency can be test-only by adding the % "test" at end.

Reply:
Looks like src/main schemas still reference the JFIF schema. Do we want to change that, so the default schema just treats al payloads as blobs, or keep the behavior of using the JPEG schema for JPEG payloads? Now a user can easily override the payload if they want jpeg support, and are src/test could add an overrid for that.